mobs/nakki_v2/ai_states/NakkiIdleState.cs

```diff
@@ -0,0 +1,81 @@
+using Godot;
+using Godot.Collections;
+using System.Linq;
+
+using Jakojaannos.WisperingWoods.Util;
+using Jakojaannos.WisperingWoods.Util.Editor;
+
+namespace Jakojaannos.WisperingWoods;
+
+[Tool]
+[GlobalClass]
+public partial class NakkiIdleState : NakkiAiState {
+	[Export] public Array<NakkiAiState> PickOneOfTheseStatesWhenDoneIdling { get; set; } = [];
+	[Export] public float IdleTime { get; set; } = 2.0f;
+	[Export] public float IdleTimeVariation { get; set; } = 0.5f;
+
+	[Export]
+	[ExportGroup("Prewire")]
+	[MustSetInEditor]
+	public NakkiStalkState StalkState {
+		get => this.GetNotNullExportPropertyWithNullableBackingField(_stalkState);
+		set => this.SetExportProperty(ref _stalkState, value);
+	}
+	private NakkiStalkState? _stalkState;
+
+	[Export]
+	[MustSetInEditor]
+	public NakkiAttackState? AttackState {
+		get => this.GetNotNullExportPropertyWithNullableBackingField(_attackState);
+		set => this.SetExportProperty(ref _attackState, value);
+	}
+	private NakkiAttackState? _attackState;
+
+	private bool _isDoneIdling = false;
+	private RandomNumberGenerator _rng = new();
+
+	public override string[] _GetConfigurationWarnings() {
+		var warnings = base._GetConfigurationWarnings() ?? [];
+
+		if (PickOneOfTheseStatesWhenDoneIdling.Count == 0) {
+			warnings = warnings.Append("Add one or more states to 'pick one of these when done'-list").ToArray();
+		}
+
+		return warnings
+			.Union(this.CheckCommonConfigurationWarnings())
+			.ToArray();
+	}
```

Comment:
With the recent changes to `CheckCommonConfigurationWarnings` and all the bumps to .NET versions etc, this could now be written as:
```suggestion
	public override string[] _GetConfigurationWarnings() {
		var warnings = PickOneOfTheseStatesWhenDoneIdling.Count == 0
			? ["Add one or more states to 'pick one of these when done'-list"]
			: Arrays.Empty<string>();
		}

		return [
			.. this.CheckCommonConfigurationWarnings(base._GetConfigurationWarnings()),
			.. warnings,
		];
	}
```